app/serializers/api/v1/order_serializer.rb

```diff
@@ -34,40 +34,24 @@ def unread_messages_count__sql
       "(select count(*) from subscriptions s where s.order_id = orders.id and s.state = 'unread' and s.user_id = orders.created_by_id)"
     end
 
-    def user_submitted_order_count
-      Order.users_order_count(object.created_by_id, 'submitted')
+    def submitted_count
+      object.submitted_count
     end
 
-    def user_submitted_order_count__sql
-      "(select count(*) from orders where orders.state = 'submitted'
-        and orders.created_by_id = orders.created_by_id)"
+    def cancelled_count
+      object.cancelled_count
```

Comment:
given these are now attributes on the order, do you need to have a method defined or is `attributes :cancelled_count` enough?

Reply:
ignore this comment in light of my above suggestion. if it works, we can remove the counts from the serializer altogether